src/appkit.rs

```diff
@@ -282,6 +288,10 @@ impl NSApplication for id {
         msg_send![self, setServicesMenu:menu]
     }
 
+    unsafe fn setWindowsMenu_(self, menu: id) {
+        msg_send![self, setWindowsMenu:menu]
```

Comment:
Properties are just sugar for methods in Objective-C. Typically a property `foo` desugars into `foo` and `setFoo`, though this can be customized. So I suspect this actually *is* using the new property.